Xcode/pkg-support/resources/CMake/sdl2_image-config.cmake

```diff
@@ -0,0 +1,49 @@
+# SDL2_image CMake configuration file:
+# This file is meant to be placed in Resources/CMake of a SDL2_image framework
+
+# INTERFACE_LINK_OPTIONS needs CMake 3.12
+cmake_minimum_required(VERSION 3.12)
+
+set(SDL2_image_FOUND TRUE)
+
+set(SDL2IMAGE_AVIF  FALSE)
+set(SDL2IMAGE_BMP   TRUE)
+set(SDL2IMAGE_GIF   TRUE)
+set(SDL2IMAGE_JPG   TRUE)
+set(SDL2IMAGE_JXL   FALSE)
+set(SDL2IMAGE_LBM   TRUE)
+set(SDL2IMAGE_PCX   TRUE)
+set(SDL2IMAGE_PNG   TRUE)
+set(SDL2IMAGE_PNM   TRUE)
+set(SDL2IMAGE_QOI   TRUE)
+set(SDL2IMAGE_SVG   TRUE)
+set(SDL2IMAGE_TGA   TRUE)
+set(SDL2IMAGE_TIF   TRUE)
```

Comment:
I believe this is FALSE

Reply:
I got it from here:
https://github.com/libsdl-org/SDL_image/blob/01a2a971dd8b48e0b63dd809ee7aa1e899486ac6/Xcode/SDL_image.xcodeproj/project.pbxproj#L649

Reply:
I got it from here:
https://github.com/libsdl-org/SDL_image/blob/01a2a971dd8b48e0b63dd809ee7aa1e899486ac6/Xcode/SDL_image.xcodeproj/project.pbxproj#L649

Reply:
Oh, sorry, I thought this was the Visual Studio support file. That's correct.